objects/rct2tt/footpath_railings/rct2tt.footpath_railings.circuitboard_invisible/object.json

```
@@ -16,7 +16,7 @@
     ],
     "strings": {
         "name": {
-            "en-GB": "Invisible railings with circuitboard queue banner",
+            "en-GB": "Sky Walk Railings",
```

I’d hesitate to call these railings, considering they _lack_ railings and only contain a support directly under the path.

Shall we just go for 
```suggestion
            "en-GB": "Sky Walk",
```
?